tools/attrd_updater.c

```diff
@@ -298,9 +314,22 @@ main(int argc, char **argv)
         goto done;
     }
 
+    if (options.attr_pattern) {
+        if (!pattern_used_correctly()) {
+            exit_code = CRM_EX_USAGE;
+            g_set_error(&error, PCMK__EXITC_ERROR, exit_code,
+                        "Error: pattern can only be used with delete, query, or update");
+            goto done;
+        }
+
+        g_free(options.attr_name);
```

Comment:
Maybe we should error if both are set

Reply:
Added a commit on top to fix this both here and in crm_attribute.